ibtk/src/utilities/fortran/Makefile.am

```diff
@@ -0,0 +1,17 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (c) 2011 - 2019 by the IBAMR developers
+## All rights reserved.
+##
+## This file is part of IBAMR.
+##
+## IBAMR is free software and is distributed under the 3-clause BSD
+## license. The full text of the license can be found in the file
+## COPYRIGHT at the top level directory of IBAMR.
+##
+## ---------------------------------------------------------------------
+
+## Process this file with automake to produce Makefile.in
+include $(top_srcdir)/config/Make-rules
+
+EXTRA_DIST = averaging.f.m4
```

Comment:
We also need `BUILT_SOURCES` and `CLEANFILES` here - take a look at one of the other fortran automake files (like math/fortran/Makefile.am)